sympy/functions/elementary/trigonometric.py

```diff
@@ -2333,9 +2333,6 @@ class acos(InverseTrigonometricFunction):
 
     A purely imaginary argument will be rewritten to asinh.
 
-    Examples
-    ========
-
```

Comment:
The other examples is the one that should be removed. The other text here isn't part of the examples.